packages/strapi/lib/middlewares/parser/index.js

```diff
@@ -40,14 +42,13 @@ module.exports = strapi => {
         // disable for graphql
         // TODO: find a better way later
         if (ctx.url === '/graphql') return next();
-
         return body({
           patchKoa: true,
-          ...strapi.config.middleware.settings.parser,
+          ...omit(strapi.config.middleware.settings.parser, 'queryStringParser'),
         })(ctx, next);
       });
 
-      addQsParser(strapi.app);
+      addQsParser(strapi.app, strapi.config.middleware.settings.parser.queryStringParser);
```

Comment:
can you replace the strapi.config.middleware.settings.parser.... access with  `strapi.config.get(...)`